packages/phone-number-privacy/signer/src/domain/quota/domainQuota.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { KnownDomain } from '@celo/identity/lib/odis/domains'
+import { Transaction } from 'knex'
+import { DomainState } from '../../database/models/domainState'
+
+export interface IDomainQuotaService {
+  doesHaveRemainingQuota: (domain: KnownDomain, domainState: DomainState) => Promise<boolean>
```

Comment:
Because quota may not be a count for all domains, I might suggest renaming this function.
```suggestion
  hasSufficientQuota: (domain: KnownDomain, domainState: DomainState) => Promise<boolean>
```